pkg/security/config/config.go

```diff
@@ -224,9 +227,11 @@ func NewConfig(cfg *config.Config) (*Config, error) {
 		EnvsWithValue:                      coreconfig.Datadog.GetStringSlice("runtime_security_config.envs_with_value"),
 
 		// runtime compilation
-		RuntimeCompilationEnabled:       coreconfig.Datadog.GetBool("runtime_security_config.runtime_compilation.enabled"),
-		RuntimeCompiledConstantsEnabled: coreconfig.Datadog.GetBool("runtime_security_config.runtime_compilation.compiled_constants_enabled"),
-		RuntimeCompiledConstantsIsSet:   coreconfig.Datadog.IsSet("runtime_security_config.runtime_compilation.compiled_constants_enabled"),
+		RuntimeCompilationEnabled:            coreconfig.Datadog.GetBool("runtime_security_config.runtime_compilation.enabled"),
+		RuntimeCompiledConstantsEnabled:      coreconfig.Datadog.GetBool("runtime_security_config.runtime_compilation.compiled_constants_enabled"),
+		RuntimeCompiledConstantsIsSet:        coreconfig.Datadog.IsSet("runtime_security_config.runtime_compilation.compiled_constants_enabled"),
+		NetworkProcessEventMonitoringEnabled: coreconfig.Datadog.GetBool("event_monitoring_config.network_process.enabled"),
```

Comment:
nit: add empty line because these are not related to runtime compilation
```suggestion
		
		NetworkProcessEventMonitoringEnabled: coreconfig.Datadog.GetBool("event_monitoring_config.network_process.enabled"),
```